packages/backend/src/loginTargetView/loginTargetView.ts

```diff
@@ -223,16 +223,18 @@ async function applyTarget(org: string, space: string) {
   }
 }
 
-function openPasscodeLink(url: string) {
-  void vscode.commands.executeCommand("vscode.open", vscode.Uri.parse(url));
+function openPasscodeLink(endpoint: string) {
+  void vscode.commands.executeCommand("vscode.open", vscode.Uri.parse(calculatePasscodeUrl(endpoint)));
 }
 
 export async function invokeLongFunctionWithProgressForm(longFunction: Function, ...args: any): Promise<any> {
   await _rpc.invoke("setBusyIndicator", [true]);
   try {
     /* eslint-disable-next-line @typescript-eslint/no-unsafe-argument */
     const ret = await longFunction(...args);
-    await _rpc.invoke("setBusyIndicator", [false]);
+    if (!["cfGetTarget", "cfGetAvailableOrgs"].includes(longFunction.name)) {
+      await _rpc.invoke("setBusyIndicator", [false]);
```

Comment:
busyIndicator is set, but it not removed always. is it as intended ?

Reply:
busyIndicator is set to True until the last command set it to False. this is not a counter here so if busyIndicator gets a True value more than one time before it gets a False value it doesn't matter